letsencrypt-win-simple/ScheduledRenewal.cs

```diff
@@ -12,6 +12,7 @@ public class ScheduledRenewal
         public string KeepExisting { get; set; }
         public string Script { get; set; }
         public string ScriptParameters { get; set; }
+        public bool Warmup { get; set; }
```

Comment:
It won't actually be able to save it as a boolean to the Registry, it should convert it to a string when it saves it, but it will then have issues converting the string back into a boolean when a renewal runs.

Take a look at KeepExisting above.